app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -79,6 +79,32 @@ def metrics_connect
                                      authentication_password('default'))
     end
 
+    def run_generic_operation(operation, path)
```

Comment:
We can make changes for the 0.3.0 hawkular-client-gem - @Jiri-Kremser is here da man :-) 
As 0.3.0 introduces a good deal of other changes, I don't want to introduce it right now.